### ISSUE TITLE
[Defect] HTML Accessibility - FAILED CHECKLIST Items as screenshots

### ISSUE THREAD

**Issue by Vyshantha** [Vyshantha/multiscripteditor#9]
https://github.com/Vyshantha/multiscripteditor/tree/main/accessibility_icons
